express-hello-world/server.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express();
+
+app.listen(8080, () => {
```

Comment:
By convention, we put this one last. First set up the middleware, then start listening.